docs/releases/1.9.txt

```diff
@@ -423,6 +423,14 @@ Miscellaneous
   Therefore, the ``@skipIfCustomUser`` decorator is no longer needed to
   decorate tests in ``django.contrib.auth``.
 
+* :meth:`FormMixin.get_context_data <django.views.generic.edit.FormMixin.get_context_data>`
```

Comment:
This note goes in backwards incompatible changes, not deprecated features. Some edits:
```
* Form initialization was moved from the :meth:`ProcessFormView.get()
  <django.views.generic.edit.ProcessFormView.get>` method to the new
  :meth:`FormMixin.get_context_data()
  <django.views.generic.edit.FormMixin.get_context_data>` method. This may be
  backwards incompatible if you have overridden the ``get_context_data()``
  method without calling ``super()``.
```